### PR TITLE
Add more VRs support

### PR DESCRIPTION
**Pull request by pchoisel**
Add support for replacing and emptying VRs: AS, DS, DT, FS, FD, IS, LS, LT, SL, SS, ST, TM, UL, UN, US, UT.

### CODE REVIEW

File: dicomanonymizer/simpledicomanonymizer.py [hunk 1]
```diff
@@ -69,42 +82,44 @@ def replace_element_date_time(element):
...
-    elif element.VR == 'IS':
+    elif element.VR in ('DS', 'IS'):
         element.value = '0'
-    elif element.VR in ('FD', 'FL', 'SS', 'US'):
+    elif element.VR in ('FD', 'FL', 'SS', 'US', 'SL', 'UL'):
         element.value = 0
-    elif element.VR == 'ST':
-        element.value = ''
+    elif element.VR in ('DT', 'DA', 'TM'):
+        handle_data_time_element(element)
```
Comment (finetjul) [inline dicomanonymizer/simpledicomanonymizer.py:116]: rename into `replace_data_time_element`

**Review by finetjul — APPROVED** [review]
LGTM

File: dicomanonymizer/simpledicomanonymizer.py (hunk 1 above)
Comment (pchoisel) [reply dicomanonymizer/simpledicomanonymizer.py]: `data` or `date` ?

Comment (finetjul) [reply dicomanonymizer/simpledicomanonymizer.py]: date

**Pull request merged**